Clicker/SectionControllers/GroupControlsSectionControllers/PollSettingsSectionController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  PollSettingsSectionController.swift
+//  Clicker
+//
+//  Created by Mathew Scullin on 3/12/19.
+//  Copyright © 2019 CornellAppDev. All rights reserved.
+//
+
+import IGListKit
+
+class PollSettingsSectionController: ListSectionController {
+    
+    // MARK: - Data vars
+    var settingsModel: PollsSettingModel!
+    
+    // MARK: - Constants
+    let cellHeight: CGFloat = 96
+    let horizontalPadding: CGFloat = 16
```

Comment:
If horizontal and vertical padding are the same, is it better convention to simply have one padding constant?

Reply:
i agree, you can just call it `padding` or something